src/qz/auth/Certificate.java

```diff
@@ -147,18 +145,16 @@ public Certificate(String in) throws CertificateParsingException {
 
             //Strip beginning and end
             String[] split = in.split("--START INTERMEDIATE CERT--");
-            byte[] serverCertificate = Base64.decode(split[0].replaceAll(X509Constants.BEGIN_CERT, "").replaceAll(X509Constants.END_CERT, ""));
 
             X509Certificate theIntermediateCertificate;
             if (split.length == 2) {
-                byte[] intermediateCertificate = Base64.decode(split[1].replaceAll(X509Constants.BEGIN_CERT, "").replaceAll(X509Constants.END_CERT, ""));
-                theIntermediateCertificate = (X509Certificate)cf.generateCertificate(new ByteArrayInputStream(intermediateCertificate));
+                theIntermediateCertificate = (X509Certificate)cf.generateCertificate(new StringBufferInputStream(split[1]));
```

Comment:
According to Oracle, this should be completely valid and return either a casted `X509Certifcate` or raise a `CertificateException`.  In reality, the library fails to parse the cert without `\r` and or `\n` per http://www.doublecloud.org/2014/03/reading-x-509-certificate-in-java-how-to-handle-format-issue/.  I will be reverting to the Base64 code to avoid any backwards compatibility issues.